ParseUI/ParseUI.xcodeproj/project.pbxproj

```diff
@@ -2033,9 +2033,11 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MARKETING_VERSION = 1.18.0;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				SKIP_INSTALL = YES;
-				TARGETED_DEVICE_FAMILY = "1,2";
+				SUPPORTS_MACCATALYST = NO;
+				TARGETED_DEVICE_FAMILY = 1;
```

Comment:
This suggests we only support iPhone. Should this be 1, 2? (This is changed from 1, 2... why?)